torch/csrc/jit/serialization/export_module.cpp

```diff
@@ -463,6 +463,8 @@ void ScriptModuleSerializer::writeArchive(
   // Vector to capture the run-time class types during pickling the IValues
   std::vector<c10::ClassTypePtr> memoizedClassTypes;
   std::vector<std::string> tensor_names;
+  // tensors which need their storages written in tensor_cdata_naming_scheme
+  std::unordered_set<std::string> tensors_to_serialize;
```

Comment:
nit: since the only time we are using this is to check that something is *not* in it, can we invert this and make it `already_serialized`.